gammapy/spectrum/tests/test_models.py

```diff
@@ -214,7 +252,7 @@ def test_models(spectrum):
         assert_quantity_allclose(model.e_peak, spectrum["e_peak"], rtol=1e-2)
 
     # inverse for TableModel is not implemented
```

Comment:
I think this is statement is not true: https://docs.gammapy.org/0.12/api/gammapy.spectrum.models.TableModel.html#gammapy.spectrum.models.TableModel.inverse. 

Maybe just add the test?

Reply:
OK. Done, it works.